docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -131,7 +123,7 @@ abstract class Client {
   public abstract on("notify_notification", (notification: NotifyNotificationRecord, metadata: Metadata) => {}): void;
 
   // Listen for when an existing notification has been changed
-  public abstract on("notify_notification_changed", (notification: NotifyNotificationRecord, metadata: Metadata) => {}): void;
+  public abstract on("notify_notifications_changed", (notifications: Record<string, { read: boolean }>) => {}): void;
```

Comment:
Why not return whole NotifyNotificationRecord instead? If we ever add any more data/states to this event we would have to have a migration

Reply:
It shows that the only thing that can be changed about a notification is its read status. If we add more state later on, we can update the specs no?